docs/tutorial/conversion.rst

```diff
@@ -296,4 +296,207 @@ Create the database and start the application::
+thus modify this function to make it use Fedora Messaging instead of fedmsg.
+
+Configuration
+-------------
+First we need to convert configuration file from fedmsg format to Fedora
+Messaging. This could be more difficult, when converting complex consumer.
+
+In case of ``the-new-hotness`` we are listening to three topics. So we
+create a new binding for them in config.toml::
+
+     [[bindings]]
+     queue = "the-new-hotness"
+     exchange = "amq.topic"
+     routing_keys = [
+        "org.release-monitoring.prod.anitya.project.version.update",
+        "org.release-monitoring.prod.anitya.project.map.new",
+        "org.fedoraproject.prod.buildsys.task.state.change",
+     ]
+
+And define a queue::
+
+    [queues.the-new-hotness]
+    durable = true
+    auto_delete = false
+    exclusive = false
+    arguments = {}
+
+Next step is to define the callback function in configuration::
+
+   callback = "hotness.consumers:BugzillaTicketFiler"
+
+This will allow you to call only ``fedora-messaging consume`` without explicitly
+specifying the callback every time you starting ``the-new-hotness``.
+
+Any application specific configuration should go to ``consumer_config`` section.
+
+Init method
+-----------
+The ``BugzillaTicketFiler`` class in ``consumers.py`` is doing all the consuming work.
+First we need to change the inheritance of this class. Fedora Messaging consumer class
+needs to inherit only from ``object``.
+
+Than we need to modify the ``__init__`` method and use the ``fedora_messaging.config.conf``
+instead of fedmsg configuration. The ``__init__`` method could look something like this
+after change::
+
+    from fedora_messaging.config import conf
+
+    class BugzillaTicketFiler(object):
+    """
+    A fedora-messaging consumer that is the heart of the-new-hotness.
+
+    This consumer subscribes to the following topics:
+
+    * 'org.fedoraproject.prod.buildsys.task.state.change'
+      handled by :method:`BugzillaTicketFiler.handle_buildsys_scratch`
+
+    * 'org.release-monitoring.prod.anitya.project.version.update'
+      handled by :method:`BugzillaTicketFiler.handle_anitya_version_update`
+
+    * 'org.release-monitoring.prod.anitya.project.map.new'
+      handled by :method:`BugzillaTicketFiler.handle_anitya_map_new`
+    """
+
+    def __init__(self):
+
+        # This is just convenient.
+        self.config = conf["consumer_config"]
+
+        self.bugzilla = hotness.bz.Bugzilla(
+            consumer=self, config=self.config["bugzilla"]
+        )
+        self.buildsys = hotness.buildsys.Koji(consumer=self, config=self.config["koji"])
+
+        default = "https://pagure.io/releng/fedora-scm-requests"
+        self.repo_url = self.config.get("repo_url", default)
+        default = "https://pdc.fedoraproject.org"
+        self.pdc_url = self.config.get("pdc_url", default)
+        default = "https://src.fedoraproject.org"
+        self.dist_git_url = self.config.get("dist_git_url", default)
```

Comment:
A lot of this stuff is specific to hotness and not related to messaging at all. It might be easier to follow if you removed anything not related to the messaging conversion (with a note that unrelated code is removed) so that readers who aren't familiar with the-new-hotness can quickly see what changes and what's relevant.